.github/policies/eventResponder.yml

```diff
@@ -192,3 +192,19 @@ configuration:
         then:
           - addLabel:
               label: "Type: Security Bug :lock:"
+
+      - description: "ITA25 - If a new PR is opened, inform module owners about the usage of the Needs Core Team label"
+        if:
+          - or:
+              - payloadType: Pull_Request
+          - isAction:
+              action: Opened
+        then:
+          - addReply:
+              reply: |
+                > [!IMPORTANT]
+                > If this is a module-related PR, being submitted by the sole owner of the module, the AVM core team must review and approve it (as module owners can't approve their on PRs).
+                >
+                > **To indicate this PR needs the core team''s attention, apply the "Needs: Core Team 🧞" label!**
```

Comment:
```suggestion
                > **To indicate this PR needs the core team's attention, apply the "Needs: Core Team 🧞" label!**
```

Reply:
This is not a mistake! It needs to be a double '' to escape this character in the YAML template of the GH policy service.